### PR TITLE
Created the logo and Navbar

### PR DESCRIPTION
**Pull request by Wenuka19**
Addresses issue #23

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```
Comment (Malith-19) [inline src/components/Navbar.js]: we don't need to import react right?

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <img src="../LogoMain.png" alt="Logo" style={{ height: '160px' }} />
+         </IconButton>
+          
+          <Button variant="contained" color='error'>Log Out</Button>
+        </Toolbar>
+      </Container>
+    </AppBar>
+  );
+}
+export default NavBar
```
Comment (Malith-19) [inline src/components/Navbar.js]: Add EoL

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            <IconButton color="inherit" aria-label="logo" sx={{marginLeft:'30px'}}>
+            <img src="../LogoMain.png" alt="Logo" style={{ height: '160px' }} />
+         </IconButton>
+          <Button variant="contained" color='error'>Log Out</Button>
+        </Toolbar>
+      </Container>
+    </AppBar>
+  );
+}
+export default NavBar;
```
Comment (nimsara66) [inline src/components/Navbar.js]: Fix EoL

Comment (Wenuka19) [reply src/components/Navbar.js]: Fixed it. Please check

**Review by Malith-19 — APPROVED** [review]

**Review by nimsara66 — APPROVED** [review]

**Pull request merged**